examples/BuddyMobileNetV3/buddy-mobilenetv3-main.cpp

```diff
@@ -1,4 +1,4 @@
-//===- MobileNetBenchmark.cpp ---------------------------------------------===//
+//===- buddy-mobilenetv3-main.cpp ---------------------------------------------===//
```

Comment:
This line of comment cannot exceed 80 columns. Please remove a few dashes to align with the comment separator below.